modules/detect_target/detect_target_brightspot.py

```diff
@@ -0,0 +1,140 @@
+"""
+Detects bright spots generated by an IR camera.
+"""
+
+import time
+
+import cv2
+import numpy as np
+
+from . import base_detect_target
+from .. import image_and_time
+from .. import detections_and_time
+from ..common.modules.logger import logger
+
+
+BRIGHTSPOT_PERCENTILE = 99.9
+DETECTION_LABEL = 1
+# Confidence set to 1 by default because SimpleBlobDetector does not compute a reponse value during keypoint detection
```

Comment:
Explain that `SimpleBlobDetector` is a binary detector, so a detection has confidence 1.0 by default.